### PR TITLE
Fix examples with react-map-gl alpha

### PR DESCRIPTION
**Pull request by balthazar**
Related to #1246

### CODE REVIEW

**Comment by balthazar** [comment]
Same segfault issue

**Comment by balthazar** [comment]
Retrying 4 times, it wooorks

**Comment by ibgreen** [comment]
Do we need the mapbox css in every example? Or is that only if we use react-map-gl markers and such?

**Comment by balthazar** [comment]
I believe we do, otherwise mapbox creates a div in the body which breaks the layer/map sync. @Pessimistress  can confirm

**Comment by 1chandu** [comment]
Does `npm run start` and `npm run start-local` both work? If not that can be addressed in another PR too.

File: examples/without-map/index.html [hunk 1]
```diff
@@ -3,6 +3,8 @@
   <head>
     <meta charset='UTF-8' />
     <title>deck.gl example</title>
+    <link href='https://api.tiles.mapbox.com/mapbox-gl-js/v0.42.2/mapbox-gl.css' rel='stylesheet' />
```
Comment (ibgreen) [inline examples/without-map/index.html]: Why does an example called `without-map` have mapbox dependencies

File: examples/svg-interoperability/index.html [hunk 2]
```diff
@@ -3,6 +3,8 @@
   <head>
     <meta charset='UTF-8' />
     <title>interoperability-demo</title>
+    <link href='https://api.tiles.mapbox.com/mapbox-gl-js/v0.42.2/mapbox-gl.css' rel='stylesheet' />
```
Comment (ibgreen) [inline examples/svg-interoperability/index.html]: @gnavvy: Does this example use react-map-gl?

File: examples/point-cloud-ply/index.html [hunk 3]
```diff
@@ -3,6 +3,8 @@
   <head>
     <meta charset='UTF-8' />
     <title>point-cloud-example</title>
+    <link href='https://api.tiles.mapbox.com/mapbox-gl-js/v0.42.2/mapbox-gl.css' rel='stylesheet' />
```
Comment (ibgreen) [inline examples/point-cloud-ply/index.html]: @gnavvy I don't think this example uses react-map-gl

File: examples/point-cloud-laz/index.html [hunk 4]
```diff
@@ -3,6 +3,8 @@
   <head>
     <meta charset='UTF-8' />
     <title>point-cloud-example</title>
+    <link href='https://api.tiles.mapbox.com/mapbox-gl-js/v0.42.2/mapbox-gl.css' rel='stylesheet' />
```
Comment (ibgreen) [inline examples/point-cloud-laz/index.html]: @gnavvy I don't think this example uses react-map-gl

File: examples/plot/index.html [hunk 5]
```diff
@@ -3,6 +3,8 @@
   <head>
     <meta charset='UTF-8' />
     <title>deck.gl example</title>
+    <link href='https://api.tiles.mapbox.com/mapbox-gl-js/v0.42.2/mapbox-gl.css' rel='stylesheet' />
```
Comment (ibgreen) [inline examples/plot/index.html]: I don't think this example uses react-map-gl

File: examples/graph/index.html [hunk 6]
```diff
@@ -4,6 +4,7 @@
     <meta charset='UTF-8' />
     <title>deck.gl example: Graph Layer</title>
     <link href="app.css" rel="stylesheet">
+    <link href='https://api.tiles.mapbox.com/mapbox-gl-js/v0.42.2/mapbox-gl.css' rel='stylesheet' />
```
Comment (ibgreen) [inline examples/graph/index.html]: Not sure this uses react-map-gl

**Pull request merged**